std/typecons.d

```diff
@@ -2261,21 +2261,20 @@ if (is(S == struct))
         }
 
         // must not escape when S is immutable
-        private
-        ref trustedPayload() @trusted
+        private ref trustedPayload() @trusted
         {
-            return *cast(S*)mutPayload.ptr;
+            return *cast(S*) mutPayload.ptr;
         }
 
         static if (!is(S == immutable))
-        ref S Rebindable_getRef() @property
+        private ref S Rebindable_getRef() @property
```

Comment:
You can't use `private` for an `alias this` symbol, because it won't be accessible outside this module. Ditto below.